script.js

```diff
@@ -1,4 +1,150 @@
+/*Requirements
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
+
+// Objective: Decide who wins the dice rolls by having the higher combined number
+//How to get there? Input roll dice function
+
+// Game modes
+var GAME_MODE_DICE_ROLL = "GAME_MODE_DICE_ROLL";
+var GAME_MODE_CHOOSE_DICE_ORDER = "GAME_MODE_CHOOSE_DICE_ORDER";
+
+// Initialise the game to start with the dice roll game mode
+var gameMode = GAME_MODE_DICE_ROLL;
+
+// Keep track of the current player's number, either 1 or 2.
+// The game starts with Player 1.
+var currPlayer = 1;
+
+// Keep track of each player's dice rolls
+var player1Dice = [];
+var player2Dice = [];
+
+// Keep track of each player's chosen numbers
+var player1Num;
+var player2Num;
```

Comment:
nice and clear global variables, well commented. i would say that if the variable name is descriptive enough, such as the case of `player1Dice`, the comment is unnecessary. this is another point for why descriptive variable names are desired as your code becomes self-documenting

i like that you're using scream case for the first two variable to denote that they won't change. Consider using the `const` keyword to declare them too rather than `var`. For example `const GAME_MODE_DICE_ROLL = "GAME_MODE_DICE_ROLL"`. I know this wasn't really covered in the course but it's a minor step up if you're interested.

For more info:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const